.gitignore

```diff
@@ -1,8 +1,8 @@
 # Deployment
 .env*
-**/data/certbot/
-**/data/nginx/app.conf
-
+**/data
```

Comment:
Added that because the embeddings saves the in the working directory where the its run. And Ifigured we don't want any data to be saved.